File: app/src/main/java/org/stratum0/statuswidget/FirstRunActivity.java
package org.stratum0.statuswidget;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.view.Window;
import android.widget.Button;

/**
 * Created by Matthias Uschok <dev@uschok.de> on 2013-10-09.
 */
public class FirstRunActivity extends Activity implements View.OnClickListener {
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        getWindow().requestFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.first_run);

        Button dismissButton = (Button) findViewById(R.id.dismissButton);
        dismissButton.setOnClickListener(this);
    }

    @Override
    public void onClick(View view) {
        if(view.getId() == R.id.dismissButton) {
            this.finish();
        }
    }
}

File: app/src/main/java/org/stratum0/statuswidget/FirstRunActivity.kt
package org.stratum0.statuswidget

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.view.View.OnClickListener
import android.view.Window
import android.widget.Button

class FirstRunActivity : Activity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        overridePendingTransition(0, 0)

        setFinishOnTouchOutside(true)
        setContentView(R.layout.first_run)

        findViewById<View>(R.id.button_dismiss).setOnClickListener { finish() }
    }

    override fun finish() {
        super.finish()

        overridePendingTransition(0, 0)
    }
}